Graph/GraphBypassing/main.py

```diff
@@ -0,0 +1,106 @@
+def readFile(array):
+    with open('input.txt') as file:
+        for line in file:
+            array.append(line.split())
+
+        for item in range(len(array)):
+            array[item][0] = int(array[item][0])
+            array[item][1] = int(array[item][1])
+
+
+def creatingArray(m, n):
+    array = [[0 for j in range(n)] for i in range(m)]
+    return array
+
+
+def creatingDictionary(array, m):
+    dictionary = {}
+    for i in range(m):
+        arr = []
+        for j in range(m):
+            if array[i][j] == 1:
+                arr.append(j+1)
+        dictionary[i+1] = arr
+    return dictionary
+
+
+def creatingAdjency(array, date, typeGraph):
+    for index in date:
+        if typeGraph == 1:
+            array[index[0] - 1][index[1] - 1] = 1
+        else:
+            array[index[0] - 1][index[1] - 1] = 1
+            array[index[1] - 1][index[0] - 1] = 1
+
+
+def iteractive_dfs(graph):
+    start = int(input("Input start top: "))
+    path, q, counter = [start], [start], 1
+    print(f'DFS-номер: {counter} \t Вершина: {q[-1]} \t Стек: {q}')
+    while q:
+        v = q[-1]
```

Comment:
And `else` clause will be unnecessary 
```suggestion
        v = q.pop()
```